### ISSUE TITLE
Online documentation links to (non-accessible) local source files

### ISSUE THREAD

**Issue by capital-G** [supercollider/supercollider#5157]
## Environment

* SuperCollider version: - 
* Operating system: - 
* Other details (Qt version, audio driver, etc.): -

## Steps to reproduce

Click on `Source: LanguageConfig.sc` @  http://doc.sccode.org/Classes/LanguageConfig.html which links to [file:///usr/local/share/SuperCollider/SCClassLibrary/Common/Core/LanguageConfig.sc](file:///usr/local/share/SuperCollider/SCClassLibrary/Common/Core/LanguageConfig.sc)

### Expected

Working link which directs to a highlighted version of the source code - maybe use github for this as a release is fixed to a commit?
I really enjoy the sphinx/readthedocs workflow in Python, e.g. https://coverage.readthedocs.io/en/coverage-5.2.1/api_coverage.html#coverage.Coverage
Docs are based on annotated source code and can be versioned so if you always have a matching version of your docs/source code available.

### Actual

Not even a 404 or else, my console (Vivaldi which is Chromium based) prints

```
Not allowed to load local resource: file:///usr/local/share/SuperCollider/SCClassLibrary/Common/Core/LanguageConfig.sc
```

On a sidenote a bunch of other errors are thrown which should be also fixed

![image](https://user-images.githubusercontent.com/8267062/91889810-bd14f580-ec8e-11ea-8cf0-fb450477af6a.png)

**Comment by dyfer** [comment]
AFAIU these links are meant to work only when help is browsed locally. Clicking on the link would open the source file in SuperCollider itself (not the help browser). I don't think this was ever meant to work when help is hosted on a web server. However, other devs may have different opinion on this.

**Comment by capital-G** [comment]
IMO this behaviour should be left intact but there should be a new way to build the docs for online consumption w/ source code by providing a flag during build time.

**Comment by capital-G** [comment]
This has been implemented via #6108
